sass/style-short.scss

```diff
@@ -1,5 +1,6 @@
 #logo {
-  height: 100px;
+  background-image: url(../css/images/gravity.coast.cassette.web.short.png);
```

Comment:
There is an image folder in dist/css/  ... should that not be copied?  Not be used?  Also, since gulp is creating the dist folder, where should I actually add the resource to the source?  That's the part that's confusing:  I add a .png to the SOURCE (not the dist)... so at SOME POINT, don't I have to tell gulp where it actually is before Gulp can gulp it up and put it where it wants to?  Should I create a variable for the image in the mixin?  I thought the point of the Dist / Gulp thing was to let Gulp rearrange the file names, file structures etc. in order to obscure the code and make it more efficient.  How can I link to something that hasn't been generated yet?